packages/block-editor/src/components/block-title/index.js

```diff
@@ -73,5 +74,10 @@ export default function BlockTitle( { clientId } ) {
 	if ( label !== blockType.title ) {
 		return truncate( label, { length: 35 } );
 	}
-	return blockInformation.title;
+	return (
+		<>
+			{ `${ blockInformation.title } ` }
+			<VisuallyHidden as="span">Block.</VisuallyHidden>
```

Comment:
I think concatenating text like that is local specific and we should probably be using something like: 

```js
sprintf( createInterpolateElement( __( '%s <VisuallyHidden>Block</VisuallyHidden>.' ), { VisuallyHidden: < VisuallyHidden /> } ), title )
```

thought the syntax in this suggestion is not entirely correct, since we'd need interpolation using

Reply:
Or maybe invert the interpolate and the sprintf call

Reply:
I implemented this.